extension/jaegerremotesampling/options.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2018 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package jaegerremotesampling
+
+import (
+	"time"
+)
+
+// Options holds configuration for the static sampling strategy store.
+type Options struct {
```

Comment:
if provider only takes two arguments I would just pass them directly without this extra struct